### PR TITLE
Cross mio updates to vocab.csv

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated vocab.csv as part of METBK sensor project - added sensors to SMs and a few other minor updates as noted in vocab_METBK_additions_2023-06-27_FINAL.xlsx (attached).  Also refer to the added highlighted columns in attached spreadsheet to verify metbk module heights.

[METBK_Module_Heights_RGT_2023-06-27.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/11885555/METBK_Module_Heights_RGT_2023-06-27.xlsx)

[vocab_METBK_additions_2023-06-27_FINAL.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/11885576/vocab_METBK_additions_2023-06-27_FINAL.xlsx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Look into ingesting METBK-CT recovered instrument data (Redmine #14304, #15493) [DT-180]](https://trello.com/c/sSSfBQmE/180-look-into-ingesting-metbk-ct-recovered-instrument-data-redmine-14304-15493-dt-180)

File: vocab/vocab.csv [hunk 1]
```diff
@@ -498,14 +498,28 @@ CP01CNSM-SBD11,Coastal Pioneer,Central Surface Mooring,Surface Buoy,,,,0,0
...
+CP01CNSM-SBD11-06-METBKA001,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
+CP01CNSM-SBD11-06-METBKA002,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-4,-4
+CP01CNSM-SBD11-06-METBKA003,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-4,-4
+CP01CNSM-SBD11-06-METBKA004,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-4,-4
+CP01CNSM-SBD11-06-METBKA005,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-4,-4
+CP01CNSM-SBD11-06-METBKA006,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-4,-4
+CP01CNSM-SBD11-06-METBKA007,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-5,-5
```
Comment (snwhite) [inline vocab/vocab.csv:508]: Looks good.

File: vocab/vocab.csv [hunk 2]
```diff
@@ -498,14 +498,28 @@ CP01CNSM-SBD11,Coastal Pioneer,Central Surface Mooring,Surface Buoy,,,,0,0
...
+CP01CNSM-SBD12-06-METBKA001,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
+CP01CNSM-SBD12-06-METBKA002,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-4,-4
+CP01CNSM-SBD12-06-METBKA003,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-4,-4
+CP01CNSM-SBD12-06-METBKA004,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-4,-4
+CP01CNSM-SBD12-06-METBKA005,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-4,-4
+CP01CNSM-SBD12-06-METBKA006,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-4,-4
+CP01CNSM-SBD12-06-METBKA007,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-5,-5
+CP01CNSM-SBD12-08-FDCHPA000,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Direct Covariance Flux,WHOI,DCFS,-5,-5
```
Comment (snwhite) [inline vocab/vocab.csv:522]: Looks good.

File: vocab/vocab.csv [hunk 3]
```diff
@@ -618,7 +632,14 @@ CP03ISSM-SBD11,Coastal Pioneer,Inshore Surface Mooring,Surface Buoy,,,,0,0
...
+CP03ISSM-SBD11-06-METBKA001,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
+CP03ISSM-SBD11-06-METBKA002,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-4,-4
+CP03ISSM-SBD11-06-METBKA003,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-4,-4
+CP03ISSM-SBD11-06-METBKA004,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-4,-4
+CP03ISSM-SBD11-06-METBKA005,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-4,-4
+CP03ISSM-SBD11-06-METBKA006,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-4,-4
+CP03ISSM-SBD11-06-METBKA007,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-5,-5
```
Comment (snwhite) [inline vocab/vocab.csv:642]: Should say "Inshore" not "Central"

File: vocab/vocab.csv [hunk 4]
```diff
@@ -684,7 +705,14 @@ CP04OSSM-SBD11,Coastal Pioneer,Offshore Surface Mooring,Surface Buoy,,,,0,0
...
+CP04OSSM-SBD11-06-METBKA001,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
+CP04OSSM-SBD11-06-METBKA002,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-4,-4
+CP04OSSM-SBD11-06-METBKA003,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-4,-4
+CP04OSSM-SBD11-06-METBKA004,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-4,-4
+CP04OSSM-SBD11-06-METBKA005,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-4,-4
+CP04OSSM-SBD11-06-METBKA006,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-4,-4
+CP04OSSM-SBD11-06-METBKA007,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-5,-5
```
Comment (snwhite) [inline vocab/vocab.csv:715]: Should say "Offshore" not "Central"

File: vocab/vocab.csv [hunk 5]
```diff
@@ -877,7 +905,7 @@ GA01SUMO-SBD11-03-HYDGN0000,Global Argentine Basin,Apex Surface Mooring,Surface
 GA01SUMO-SBD11-04-DOSTAD000,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Dissolved Oxygen,Aanderaa,Optode 4831,1,1
 GA01SUMO-SBD11-05-SPKIRB000,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Spectral Irradiance,Satlantic,OCR507 ICSW,-5,-5
 GA01SUMO-SBD11-06-METBKA000,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LGR,0,0
-GA01SUMO-SBD11-06-METBKA001,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,SBE 37-SM,1,1
+GA01SUMO-SBD11-06-METBKA001,Global Argentine Basin,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
```
Comment (snwhite) [inline vocab/vocab.csv:908]: Looks good.

File: vocab/vocab.csv [hunk 6]
```diff
@@ -1076,7 +1104,14 @@ GI01SUMO-SBD11-01-MOPAK0000,Global Irminger Sea,Apex Surface Mooring,Surface Buo
...
+GI01SUMO-SBD11-06-METBKA001,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
+GI01SUMO-SBD11-06-METBKA002,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-5,-5
+GI01SUMO-SBD11-06-METBKA003,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-5,-5
+GI01SUMO-SBD11-06-METBKA004,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-5,-5
+GI01SUMO-SBD11-06-METBKA005,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-5,-5
+GI01SUMO-SBD11-06-METBKA006,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-5,-5
+GI01SUMO-SBD11-06-METBKA007,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-6,-6
```
Comment (snwhite) [inline vocab/vocab.csv:1114]: Looks good.

File: vocab/vocab.csv [hunk 7]
```diff
@@ -1085,8 +1120,15 @@ GI01SUMO-SBD12-02-FLORTD000,Global Irminger Sea,Apex Surface Mooring,Surface Buo
...
+GI01SUMO-SBD12-06-METBKA001,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
+GI01SUMO-SBD12-06-METBKA002,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-5,-5
+GI01SUMO-SBD12-06-METBKA003,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-5,-5
+GI01SUMO-SBD12-06-METBKA004,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-5,-5
+GI01SUMO-SBD12-06-METBKA005,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-5,-5
+GI01SUMO-SBD12-06-METBKA006,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-5,-5
+GI01SUMO-SBD12-06-METBKA007,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-6,-6
+GI01SUMO-SBD12-08-FDCHPA000,Global Irminger Sea,Apex Surface Mooring,Surface Buoy,Direct Covariance Flux,WHOI,DCFS,-6,-6
```
Comment (snwhite) [inline vocab/vocab.csv:1131]: Looks good.

File: vocab/vocab.csv [hunk 8]
```diff
@@ -1517,7 +1559,14 @@ GS01SUMO-SBD11-03-HYDGN0000,Global Southern Ocean,Apex Surface Mooring,Surface B
...
+GS01SUMO-SBD11-06-METBKA001,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
+GS01SUMO-SBD11-06-METBKA002,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-5,-5
+GS01SUMO-SBD11-06-METBKA003,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-5,-5
+GS01SUMO-SBD11-06-METBKA004,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-5,-5
+GS01SUMO-SBD11-06-METBKA005,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-5,-5
+GS01SUMO-SBD11-06-METBKA006,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-5,-5
+GS01SUMO-SBD11-06-METBKA007,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-6,-6
```
Comment (snwhite) [inline vocab/vocab.csv:1569]: Looks good.

File: vocab/vocab.csv [hunk 9]
```diff
@@ -1526,8 +1575,15 @@ GS01SUMO-SBD12-02-FLORTD000,Global Southern Ocean,Apex Surface Mooring,Surface B
...
+GS01SUMO-SBD12-06-METBKA002,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-5,-5
+GS01SUMO-SBD12-06-METBKA003,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-5,-5
+GS01SUMO-SBD12-06-METBKA004,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-5,-5
+GS01SUMO-SBD12-06-METBKA005,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-5,-5
+GS01SUMO-SBD12-06-METBKA006,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-5,-5
+GS01SUMO-SBD12-06-METBKA007,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-6,-6
+GS01SUMO-SBD12-08-FDCHPA000,Global Southern Ocean,Apex Surface Mooring,Surface Buoy,Direct Covariance Flux,WHOI,DCFS,-6,-6
```
Comment (snwhite) [inline vocab/vocab.csv:1586]: Looks good.

**Review by snwhite — CHANGES_REQUESTED** [review]
Cut and Paster error for Pioneer Inshore and Offshore need to be corrected.

File: vocab/vocab.csv (hunk 4 above)
Comment (rtravis-whoi) [reply vocab/vocab.csv]: correction made

File: vocab/vocab.csv (hunk 3 above)
Comment (rtravis-whoi) [reply vocab/vocab.csv]: correction made

File: vocab/vocab.csv [hunk 10]
```diff
@@ -618,7 +632,14 @@ CP03ISSM-SBD11,Coastal Pioneer,Inshore Surface Mooring,Surface Buoy,,,,0,0
...
+CP03ISSM-SBD11-06-METBKA001,Coastal Pioneer,Inshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
+CP03ISSM-SBD11-06-METBKA002,Coastal Pioneer,Inshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-4,-4
+CP03ISSM-SBD11-06-METBKA003,Coastal Pioneer,Inshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-4,-4
+CP03ISSM-SBD11-06-METBKA004,Coastal Pioneer,Inshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-4,-4
+CP03ISSM-SBD11-06-METBKA005,Coastal Pioneer,Inshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-4,-4
+CP03ISSM-SBD11-06-METBKA006,Coastal Pioneer,Inshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-4,-4
+CP03ISSM-SBD11-06-METBKA007,Coastal Pioneer,Inshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-5,-5
```
Comment (snwhite) [inline vocab/vocab.csv:642]: Looks good.

File: vocab/vocab.csv [hunk 11]
```diff
@@ -684,7 +705,14 @@ CP04OSSM-SBD11,Coastal Pioneer,Offshore Surface Mooring,Surface Buoy,,,,0,0
...
+CP04OSSM-SBD11-06-METBKA001,Coastal Pioneer,Offshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
+CP04OSSM-SBD11-06-METBKA002,Coastal Pioneer,Offshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-4,-4
+CP04OSSM-SBD11-06-METBKA003,Coastal Pioneer,Offshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-4,-4
+CP04OSSM-SBD11-06-METBKA004,Coastal Pioneer,Offshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-4,-4
+CP04OSSM-SBD11-06-METBKA005,Coastal Pioneer,Offshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-4,-4
+CP04OSSM-SBD11-06-METBKA006,Coastal Pioneer,Offshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-4,-4
+CP04OSSM-SBD11-06-METBKA007,Coastal Pioneer,Offshore Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-5,-5
```
Comment (snwhite) [inline vocab/vocab.csv:715]: Looks good.

**Review by snwhite — APPROVED** [review]

File: vocab/vocab.csv (hunk 1 above)
Comment (cdobs) [inline vocab/vocab.csv:508]: Looks good

File: vocab/vocab.csv [hunk 12]
```diff
@@ -498,14 +498,28 @@ CP01CNSM-SBD11,Coastal Pioneer,Central Surface Mooring,Surface Buoy,,,,0,0
...
+CP01CNSM-SBD12-06-METBKA001,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Sea-Bird Electronics,ASIMET-SBE37SM,1,1
+CP01CNSM-SBD12-06-METBKA002,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-BPR,-4,-4
+CP01CNSM-SBD12-06-METBKA003,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-HRH,-4,-4
+CP01CNSM-SBD12-06-METBKA004,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-LWR,-4,-4
+CP01CNSM-SBD12-06-METBKA005,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-PRC,-4,-4
+CP01CNSM-SBD12-06-METBKA006,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWR,-4,-4
+CP01CNSM-SBD12-06-METBKA007,Coastal Pioneer,Central Surface Mooring,Surface Buoy,Bulk Meteorology Instrument Package,Star Engineering,ASIMET-SWND,-5,-5
```
Comment (cdobs) [inline vocab/vocab.csv:521]: Looks good

File: vocab/vocab.csv (hunk 10 above)
Comment (cdobs) [inline vocab/vocab.csv:642]: Looks good

File: vocab/vocab.csv (hunk 11 above)
Comment (cdobs) [inline vocab/vocab.csv:715]: Looks good

**Review by cdobs — APPROVED** [review]

**Pull request merged**